pkg/cluster/controllers/common/utils.go

```diff
@@ -12,12 +12,12 @@ import (
 
 func GetIngressPath(ingress v1alpha1.IngressController) (string, error) {
 	switch ingress {
-	case v1alpha1.IngressControllerDefault:
+	case
```

Comment:
Do we still need this switch case? The kb spec annotation has indicated that the controller value will be limited in the enum and open API will validate them. So, the value here should be a known controller. Am I right?

There are some similar cases above.

Reply:
yes, it can be removed